src/Microsoft.Data.SqlClient/netfx/src/Resources/Strings.de.resx

```diff
@@ -2818,7 +2818,7 @@
     <value>Der angegebene ColumnName '{0}' entspricht keiner Spalte in der Datenquelle.</value>
   </data>
   <data name="SQL_BulkLoadStringTooLong" xml:space="preserve">
-    <value>String- oder binary-Daten würden abgeschnitten.</value>
+    <value>String- oder binary-Daten würden in Tabelle '{0}', Spalte '{1}' abgeschnitten. Abgeschnittener Wert: '{2}'.</value>
```

Comment:
Please revert localization file changes, these changes will be handled internally!